src/elements/content-sidebar/activity-feed/comment/CommentDeleteConfirmation.scss

```diff
@@ -0,0 +1,21 @@
+.bcs-CommentDeleteConfirmation {
+    width: 212px;
+
+    .overlay {
+        border-radius: 4px;
+        padding: 20px;
+    }
+}
+
+.bcs-CommentDeleteConfirmation-prompt {
+    margin-bottom: 20px;
+}
+
+.bcs-CommentDeleteConfirmation-delete,
+.bcs-CommentDeleteConfirmation-cancel {
+    margin-left: 0;
+}
+
+.bcs-CommentDeleteConfirmation-cancel {
+    margin-right: 10px;
```

Comment:
Can we swap this for `margin-left` on the delete button?

Reply:
no, when the 2 buttons wrap, the right button would not be left-aligned